src/server/api/routes/users.router.js

```diff
@@ -182,4 +182,42 @@ router.get('/:id/favorites/', (req, res, next) => {
     .catch(next);
 });
 
+/**
+ * @swagger
+ * /users/{user_id}/favorites:
+ *  post:
+ *    tags:
+ *    - Users
+ *    summary: Save user favorites
+ *    description:
+ *      To Save new favorite from user to the DB
+ *    produces: application/json
+ *    parameters:
+ *     - in: body
+ *       name: user
+ *       description: create a new favorite item for user
+ *       schema:
+ *         type: object
+ *         required: true
+ *         description: user json object
+ *         properties:
+ *            user_id:
+ *              type: string
```

Comment:
The user id already comes in rouote path `users/{user_id}/favorites` 
I think you dont need to add it second time